src/App.vue

```diff
@@ -95,6 +100,11 @@ export default {
     padding-top: 10px;
     border-top: 1px solid #424242;
 }
+
+
```

Comment:
Remove blank lines

Reply:
Please have a look at the PR so that I can work on the next issues.